lib/guardian/db/sweeper.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Guardian.DB.Sweeper do
+  @moduledoc """
+  A GenServer that periodically checks for, and expires, tokens from storage.
+
+  To leverage the automated Sweeper functionality update your project's Application
+  file to include the following child in your supervision tree:
+
+  ## Example
+
+    worker(Guardian.DB.Sweeper, [interval: 60])
+  """
+  use GenServer
+
+  alias Guardian.DB.Token
+
+  @sixty_minutes 60 * 60 * 1000
+
+  def start_link(opts) do
+    interval = Keyword.get(opts, :interval, @sixty_minutes)
```

Comment:
looks like it's `: sweep_interval` in the config, @doomspork 

```
config :guardian, Guardian.DB,
  repo: MyApp.Repo, # Add your repository module
  schema_name: "guardian_tokens", # default
  token_types: ["refresh_token"], # store all token types if not set
  sweep_interval: 60 # default: 60 minutes
```

Reply:
oh I got it now, so it's not configurable anymore?

Reply:
It is configurable @alexfilatov but as I mentioned in this point:

> 4. Stop accessing configuration directly from the library code. It's generally a better practice for the application to have its own configuration which it passes in rather than libraries doing that themselves.

I think we should be moving away from our libraries accessing application configuration themselves directly. Applications should be able to pass in their configurations. 

I'm open to _not_ doing this, I think it's just a better design practice.

Reply:
I love this part for sure